bin/steps/swift-install

```diff
@@ -1,13 +1,15 @@
+DYNAMIC_LIBRARY_REGEX=".*\.so\(\.[0-9]+\)*\'"
```

Comment:
My regex skills aren't the greatest, but I think this should match `*.so`, `*.so.X`, `*.so.X.Y`, and `*.so.X.Y.Z`.